ro.go.hmlendea.Duolingo.yaml

```diff
@@ -0,0 +1,51 @@
+app-id: ro.go.hmlendea.Duolingo
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+separate-locales: false
+command: duolingo-desktop
+rename-icon: ro.go.hmlendea.Duolingo
```

Comment:
```suggestion
```

the icon already has that name.

Reply:
It fails with `Error loading desktop file: Failed to find icon: Failed to find icon duolingo` if I remove that

Reply:
I could use `desktop-file-edit` to change it to `ro.go.hmlendea.Duolingo` in the desktop file but that's pretty redundant since the `rename-icon` seems to do the same thing (I think)

Reply:
that's because when you installed the icon as `duolingo.png` and I told you to add the rename icon you ignored it. Now that you install it with the flatpak_id name, you have to fix the .desktop file as I mentioned.

You still don't.

Reply:
I didn't ignore anything. I tried to make the changes as I understood them.
Hopefully this latest commit is.

Reply:
you still did.

See THIS: https://github.com/flathub/flathub/pull/2926#discussion_r828428799

Reply:
Like this then? https://github.com/flathub/flathub/pull/2926/commits/ba5b9c9817a3048baf8263f6a71172c246bba67b